### ISSUE TITLE
Cannot be installed in qBraid since python version is backdated.

### ISSUE THREAD

**Issue by pradhole** [qiskit-community/Quantum-Challenge-Grader#172]
While installing the package `qc-grader` in qBraid system, I get the error stating that the package requires a different Python: 3.9.18 not in '>=3.10'
Can we update the python requirements for this package?

**Comment by khushi707** [comment]
Did you find any way to resolve this issue ?

**Comment by pradhole** [comment]
@khushi707  Not really. qBraid system is very glitchy. I used a python virtualenv instead. That built fairly easily.

**Comment by Eric-Arellano** [comment]
Going to close this as stale. Please feel free to open a new issue if you're having issues installing this package.
